scripts/analyze.sh

```diff
@@ -233,6 +234,11 @@ if check_for_require aio || check_config_file ZJS_AIO; then
     >&2 echo Using module: AIO
     MODULES+=" -DBUILD_MODULE_AIO"
     echo "CONFIG_ADC=y" >> arc/prj.conf.tmp
+    # Workaround for the Zephyr issue ZEP-1882: ADC doesn't work with
+    # SDK 0.9 due to some compiler optimization, so enable debug mode.
+    if [ $SDK_VERSION = "0.9" ]; then
+        echo "CONFIG_DEBUG=y" >> arc/prj.conf.tmp
```

Comment:
Could you add:
(>&2 echo Warning: Zephyr SDK 0.9 detected, compiling in debug mode!)

in here and make sure that works... I think we'll stick w/ 0.8.2 as our default until this is fixed.